### ISSUE TITLE
Additional technical review of language setting function other than Android official support language

### ISSUE THREAD

**Issue by hanjoongcho** [hanjoongcho/aaf-easydiary#137]
Reference Issue:
#136 
#59 

Reference link for implementation plan:
https://stackoverflow.com/questions/2900023/change-app-language-programmatically-in-android

Other reference links:
https://scriptsource.org/cms/scripts/page.php?item_id=script_detail&key=Olck

**Comment by hanjoongcho** [comment]
Hello @Prasanta-Hembram 
It is possible to programmatically support languages that are not supported by the Android platform, but it is not a good method.
However, it seems that Android Studio can create a Santali resource file.
If there is a function to set the language of the screenshot attached to the language setting menu of your device, I think Santali language service will be possible.
Is there a Santali language setting function in the device setting menu?
![image](https://user-images.githubusercontent.com/7098851/98005257-2ef6f000-1e34-11eb-82a2-dec4791d880e.png)

**Comment by Prasanta-Hembram** [comment]
Hi, @hanjoongcho I think it is tough to make support for Santali. You mean changing languages from device settings?? I don't think possible right now. What you told is only possible when a device ROM is made support for that language, i think. I think it is a tribal language and hence a minor language and less spoken than other languages, so it might not have been added. I don't think any device till now supports this language. Their are few but they are keypad phones.

I used Android 8 oreo in Redmi 5 device but it still doesn't support it from settings. It doesn't even support in latest MIUI 12....I contacted some of my friends.... It may be possible in future i think but not right now it will take years 🤔🤔🤔

I checked major rom players like Lineage Os, One Touch, MIUI doesn't have resources for Santali. So it us quite clear not possible:-(( .. ok then better luck next time..
